mempool/mempool.go

```diff
@@ -37,6 +38,13 @@ func (p *Pool) Enqueue(userTxn comettypes.Tx) error {
 	// Unfortunately, comet's DB interface doesn't support it.
 	// Moving to a different DB interface is left for future work.
 
+	// Attempt to adapt the Cosmos transaction to an Ethereum deposit transaction.
+	// If the adaptation results in one or more transactions, it indicates that the
+	// user transaction is a deposit transaction, which is not allowed in the pool.
```

Comment:
I think we changed the code but forgot the comment before.
```suggestion
	// If the adaptation succeeds, it indicates that the
	// user transaction is a deposit transaction, which is not allowed in the pool.
```